Source/Scene/Cesium3DTileset.js

```diff
@@ -786,6 +802,68 @@ define([
 
     ///////////////////////////////////////////////////////////////////////////
 
+    var scratchJulianDate = new JulianDate();
+
+    function updateExpireDate(tile) {
+        if (defined(tile.expireDuration)) {
```

Comment:
Ah I think I see the problem you're talking about now. If the expireData doesn't exist then it won't update.